### PR TITLE
simplify panel height logic, controlling height with css

### PR DESCRIPTION
**Pull request by akash1810**
Fixes issue of disappearing info panel too!

### CODE REVIEW

**Comment by akash1810** [comment]
tested on TEST too.

File: kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
-<div class="panel"
-     ng:init="editing = false">
-    <div class="panel__content">
+<div ng:init="editing = false">
+    <div>
```
Comment (theefer) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html]: Do we need this `div`?

**Comment by theefer** [comment]
Not quite working as is (crops the bottom of the RHS panel), maybe `min-height`? In my experience `height: 100%` never quite worked for me. @jamesgorrie may have more ideas.

File: kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html (hunk 1 above)
Comment (akash1810) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html]: nope, removed.

**Comment by akash1810** [comment]
Fixed now, using `calc`.

File: kahuna/public/js/components/gr-panels/gr-panels.css [hunk 2]
```diff
@@ -51,3 +48,7 @@
     display: block;
     margin-top: 36px;
 }
+
+.gr-panel-height {
+    height: calc(100vh - 86px); /* 50px search bar, 36px toolbar */
```
Comment (theefer) [inline kahuna/public/js/components/gr-panels/gr-panels.css]: @jamesgorrie didn't we use `calc` before? wondering why we went to JS if so.

Comment (jamesgorrie) [inline kahuna/public/js/components/gr-panels/gr-panels.css]: I think I was overcomplicating  / abstracting things - so it was obviously broken.

File: kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-        <!-- We use $parent.editing here as ng:if creates it's own child scope -->
-        <div class="flex-container collections-panel-heading text-small" ng:if="! ctrl.error">
-            <div class="flex-spacer">Collections</div>
+    <!-- TODO: we'll probably move this out to another template once we have something more
+    final -->
+    <!-- We use $parent.editing here as ng:if creates it's own child scope -->
+    <div class="flex-container collections-panel-heading text-small" ng:if="! ctrl.error">
+        <div class="flex-spacer">Collections</div>
 
             <button ng:class="{'button-edit': ! $parent.editing, 'button-save': $parent.editing}"
```
Comment (tsop14) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html]: These don't need to be indented.

**Comment by tsop14** [comment]
:+1: only indent.

File: kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html (hunk 3 above)
Comment (akash1810) [inline kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html]: good spot!

**Pull request merged**